internal/netif/netif.go

```diff
@@ -50,7 +57,21 @@ func (m *netifManagerDefault) EnsureIPAddress() error {
 
 	l, err := m.LinkByName(m.devName)
 	if err != nil {
-		return xerrors.Errorf("could not get loopback interface %v", err)
+		var linkNotFoundErr netlink.LinkNotFoundError
+		if !errors.As(err, &linkNotFoundErr) && !m.manageInterface {
+			return xerrors.Errorf("could not get interface %s: %v", m.devName, err)
+		}
+
+		attrs := netlink.LinkAttrs{
+			Name: m.devName,
+			MTU:  65535,
+		}
+		dummyLink := &netlink.Dummy{LinkAttrs: attrs}
+		err = m.LinkAdd(dummyLink)
+		if err != nil {
+			return xerrors.Errorf("could add dummy interface %v", err)
```

Comment:
Please add the name of the device to the error message and use a colon to separate the message from the error text.